arch/x86/kvm/vmx/pkvm_host.c

```diff
@@ -62,6 +86,21 @@ static int pkvm_vm_init(struct kvm *kvm)
 	return ret;
 }
 
+static void pkvm_vm_destroy(struct kvm *kvm)
+{
+	union pkvm_hc_data inout = { 0 };
```

Comment:
nit: do we need to zero-initialize `inout` (here as well as in pkvm_vcpu_free() etc)? We set the needed fields anyway, and the rest of the bytes in `inout` should not be used anyway?